### PR TITLE
test: update production tests

### PR DESCRIPTION
**Pull request by sloanelybutsurely**
runs all tests for production again

### CODE REVIEW

File: .github/workflows/automation-prod.yml [hunk 1]
```diff
@@ -18,4 +18,3 @@ jobs:
         uses: cypress-io/github-action@v4
         with:
           env: youth_pass_url=https://mbta.prod.simpligov.com/prod/portal/ShowWorkFlow/AnonymousEmbed/e5e2dba6-5424-48d6-80d9-16a27e0fdd84,senior_url=https://mbta.prod.simpligov.com/prod/portal/ShowWorkFlow/AnonymousEmbed/fc6ff5b0-f3bd-436d-8201-b6a4b98bc7fa
-          spec: cypress/automated-tests/youth-pass/address-section-required-fields.cy.js,cypress/automated-tests/youth-pass/eligibility-checker-blockers.cy.js,cypress/automated-tests/youth-pass/file-type-upload.cy.js,cypress/automated-tests/youth-pass/personal-contact-sections-required-fields.cy.js,cypress/automated-tests/youth-pass/successful-youth-pass-submission.cy.js,cypress/automated-tests/senior/birth-date-validation.cy.js,cypress/automated-tests/senior/required-fields.cy.js
```
Comment (napilla) [inline .github/workflows/automation-prod.yml:21]: I can get why we removed youth-pass but is there a reason the `automated-tests/senior` files were removed?

File: .github/workflows/automation-prod.yml [hunk 2]
```diff
@@ -18,4 +18,3 @@ jobs:
         uses: cypress-io/github-action@v4
         with:
           env: youth_pass_url=https://mbta.prod.simpligov.com/prod/portal/ShowWorkFlow/AnonymousEmbed/e5e2dba6-5424-48d6-80d9-16a27e0fdd84,senior_url=https://mbta.prod.simpligov.com/prod/portal/ShowWorkFlow/AnonymousEmbed/fc6ff5b0-f3bd-436d-8201-b6a4b98bc7fa
```
Comment (napilla) [inline .github/workflows/automation-prod.yml:20]: do we want to remove the `youth_pass_url` here too?

**Review by napilla — COMMENTED** [review]
Reviewing so we can get this along: mostly just commenting because i need more context around the changes here.

File: .github/workflows/automation-prod.yml (hunk 2 above)
Comment (sloanelybutsurely) [reply .github/workflows/automation-prod.yml:20]: no. youth pass is still tested. the tests just assert that the retirement page is shown and the "submit" button is hidden

**Review by napilla — APPROVED** [review]

**Pull request merged**